.github/workflows/test.yml

```
@@ -2,8 +2,16 @@ name: ovn-ci
 
 on:
   merge_group:
+    # Only run jobs if at least one non-doc file is changed
+    paths-ignore:
```

actionlint (1.7.7)

"paths-ignore" filter is not available for merge_group event. it is only for push, pull_request, pull_request_target events

(events)